src/webui/www/private/views/preferences.html

```diff
@@ -2393,6 +2406,7 @@
             settings.set('peer_turnover_cutoff', $('peerTurnoverCutoff').getProperty('value'));
             settings.set('peer_turnover_interval', $('peerTurnoverInterval').getProperty('value'));
             settings.set('request_queue_size', $('requestQueueSize').getProperty('value'));
+            settings.set('storage_type', $('storageType').getProperty('value'));
```

Comment:
ditto.
Code in appcontroller.cpp/session.h/session.cpp needs to be adjusted too.